components/ArrowButton/ArrowButton.stories.tsx

```diff
@@ -1,23 +1,29 @@
-import { Meta, StoryFn } from "@storybook/react";
+import { Meta, StoryObj } from "@storybook/react";
+import { userEvent, within } from "@storybook/testing-library";
 import { ArrowButton } from "./ArrowButton";
 
 export default {
   title: "Arrow button",
   component: ArrowButton,
+  argTypes: {
+    onClick: { action: "Clicked" },
```

Comment:
Hola @ErikGIovani, aca es `handleClick`, creia que juncionaba con el `onClick` tambien, pero aparentemente no..

```suggestion
    handleClick: { action: "Clicked" },
```